### DISCUSSION TITLE
Is the Modrinth client (F)OSS?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [modrinth/.github#219] [Other discussions]
I don't see a repository for it at https://github.com/orgs/modrinth/repositories?type=source.

**Comment by triphora** [comment]
Right at the top: https://github.com/modrinth/theseus

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/modrinth/discussions/219#discussioncomment-8239587

Thanks, @triphora - wasn't aware it was codenamed that. Would you mind adding the Q&A discussion preset (and converting this into one) so that I can mark your response as the answer?

**Reply by triphora** [reply]
We do not have the Q&A feature enabled because we would be forced to convert all discussions in the "Other discussions" section to that, when most do not follow the Q&A format. For the most part, questions tend to be asked and answered in [our support portal](https://support.modrinth.com).

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/modrinth/discussions/219#discussioncomment-8239758

@triphora, they can be enabled alongside "General", which all current "Other discussions" would become. It acts identically. See https://github.com/RokeJulianLockhart/rokejulianlockhart/discussions/new/choose, for instance:

> ![image](https://github.com/modrinth/.github/assets/42837531/79ca7f36-c094-49e8-91f5-94468b5f0485)
